jaxrs/pom.xml

```diff
@@ -33,15 +33,31 @@
             <artifactId>jackson-jaxrs-json-provider</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.auto.value</groupId>
+            <artifactId>auto-value</artifactId>
+        </dependency>
+        <dependency>
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
We shouldn't need to add a dependency on gson. We already have jackson pulled in, we should be able to just use that.

Reply:
Removed depdency on Gson from the parent pom: https://github.com/cerner/beadledom/pull/45/commits/5526d8c027550e6ea5657d742f4ae878f66ac4d7#diff-600376dffeb79835ede4a0b285078036L276 and the jaxrs pom: https://github.com/cerner/beadledom/pull/45/commits/5526d8c027550e6ea5657d742f4ae878f66ac4d7#diff-417137b68e0964b6529eb21151e451c8L45

Instead, using `JsonErrorMatchers` with Jackson: https://github.com/cerner/beadledom/pull/45/commits/5526d8c027550e6ea5657d742f4ae878f66ac4d7#diff-09a50fae7f92d0c897a4330007129927R1